### PR TITLE
Refreshing Upgrade plugin instruction and FAQ for upgrading to 0.13 managed plugin [CI SKIP]

### PR DESCRIPTION
**Pull request by shuklanirdesh82**
While upgrading to 0.13 managed plugin from older installation, ```docker service restart``` is required due to known docker [Issue #32635](https://github.com/docker/docker/issues/32635) 

Refreshing FAQ to accommodate such information.

/CC @msterin @govint

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -61,8 +61,10 @@ sudo rpm -e docker-volume-vsphere # Photon or rpm based distros
 
 **For Docker 1.13 and above**, install managed plugin from Docker Store.
 ```
-docker plugin install --grant-all-permissions --alias vsphere vmware/docker-volume-vsphere:latest
+1. docker plugin install --grant-all-permissions --alias vsphere vmware/docker-volume-vsphere:latest
+2. systemctl restart docker # restart docker service incase of upgrade due to known docker issue #32635
```
Comment (pdhamdhere) [inline README.md]: `systemctl restart docker` should be @ L60 with "uninstall older releases"

File: README.md [hunk 2]
```diff
@@ -61,8 +61,10 @@ sudo rpm -e docker-volume-vsphere # Photon or rpm based distros
...
 ```
+**Note**: If you are upgrading to 0.13 managed plugin, docker service needs to be restarted until [Issue #32635](https://github.com/docker/docker/issues/32635) is resolved.
```
Comment (pdhamdhere) [inline README.md]: Duplicate. Can be removed.

File: docs/user-guide/faq.md [hunk 3]
```diff
@@ -37,6 +37,33 @@ It occurs if the Docker volume service cannot communicate to the ESX back end. T
 
 It occurs if the linux kernel does not know about the AF family used for VMCI communication. Please read ["What is VMCI and vSock and why is it needed?"](https://vmware.github.io/docker-volume-vsphere/user-guide/faq/#what-is-vmci-and-vsock-and-why-is-it-needed) above.
 
+## Upgrade to version 0.13 (March 2017) release (installing vDVS plugin as Managed plugin)
```
Comment (pdhamdhere) [inline docs/user-guide/faq.md]: I would make it short for e.g.

### I'm not able to create volume after upgrading to vDVS managed plugin
```
# docker volume create -d vsphere vol5
Error response from daemon: create vol5: Post http://%2Frun%2Fdocker%2Fplugins%2Fvsphere.sock/VolumeDriver.Create: dial unix /run/docker/plugins/vsphere.sock: connect: no such file or directory
```
You are running into known [Docker Issue #32635](https://github.com/docker/docker/issues/32635). Restart docker e.g.
```
systemctl restart docker
```

File: README.md (hunk 1 above)
Comment (shuklanirdesh82) [inline README.md]: done!

File: README.md (hunk 2 above)
Comment (shuklanirdesh82) [inline README.md]: done!

File: docs/user-guide/faq.md (hunk 3 above)
Comment (shuklanirdesh82) [inline docs/user-guide/faq.md]: 👍

**Pull request merged**